src/pb_stub.cc

```diff
@@ -741,13 +741,17 @@ Stub::ProcessRequests(RequestBatch* request_batch_shm_ptr)
       }
     }
 
+
     response_batch_shm_ptr->batch_size = response_size;
 
     std::vector<std::shared_ptr<PbTensor>> gpu_tensors;
 
     size_t i = 0;
-    for (auto& response : responses) {
-      InferResponse* infer_response = response.cast<InferResponse*>();
+    for (; i < batch_size; i++) {
```

Comment:
confused why we don't just do
```suggestion
    for (size_t i = 0; i < batch_size; i++) {
```

Reply:
No, particular reason. Fixed.